producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/bulkfeedback/BulkFeedbackConfigBTwo.java

```diff
@@ -0,0 +1,257 @@
+package uk.gov.moj.sdt.producers.comx.config.bulkfeedback;
+
+import java.time.LocalDateTime;
+
+import com.google.common.collect.Lists;
+import org.springframework.beans.factory.config.MethodInvokingFactoryBean;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.moj.sdt.domain.BulkSubmission;
+import uk.gov.moj.sdt.producers.comx.utils.BulkFeedbackFactory;
+
+@Configuration
+@EnableAutoConfiguration
+public class BulkFeedbackConfigBTwo {
+
+    @Bean("createIndividualRequestB0000002")
+    public void invokingBean() {
+        MethodInvokingFactoryBean methodInvokingFactoryBean = new MethodInvokingFactoryBean();
+        methodInvokingFactoryBean.setTargetObject(bulkFeedbackFactory());
+        methodInvokingFactoryBean.setTargetMethod("createIndividualRequests");
+
+        methodInvokingFactoryBean.setArguments(Lists.newArrayList(
+            Lists.newArrayList(
+                "USER_REQUEST_ID_B1a",
+                "mcolClaim",
+                "Rejected",
+                "DUP_CUST_REQID",
+                "Duplicate User Request Identifier submitted USER_REQUEST_ID_B1a.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_B1b",
+                "mcolJudgment",
+                "Rejected",
+                "DUPLD_CUST_REQID",
+                "Unique Request Identifier has been specified more than once within the originating Bulk Request.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_B1b",
+                "mcolJudgment",
+                "Rejected",
+                "DUPLD_CUST_REQID",
+                "Unique Request Identifier has been specified more than once within the originating Bulk Request.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_B2",
+                "mcolClaim",
+                "Rejected",
+                "8",
+                "First defendant's postcode is not in England or Wales.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_B3",
+                "mcolJudgment",
+                "Rejected",
+                "75",
+                "Rejected by CCBC - rejection number 6.",
+                ""
+            ),
+            Lists.newArrayList("USER_REQUEST_ID_B4",
+                               "mcolClaimStatusUpdate",
+                               "Accepted",
+                               "",
```

Comment:
Make this consistent with the other Lists.newArrayList() by starting the parameters one a new line, i.e.
```
Lists.newArray(
    "USER_REQUEST_ID_B4",
    "mcolClaimStatusUpdate",
    "Accepted",
    "",
    ...
```
This also applies to the data being set up for all the following requests:

- USER_REQUEST_ID_B6
- USER_REQUEST_ID_B7
- USER_REQUEST_ID_B11 - USER_REQUEST_ID_B14
- USER_REQUEST_ID_B16
- USER_REQUEST_ID_B17

Reply:
done